### ISSUE TITLE
Cannot click on the following cell

### ISSUE THREAD

**Issue by igorquinta** [imbus/robotframework-robosapiens#11]
I can't click on the following cell.

I Try:

"Double-click Cell   1   Pedido"
"Push Button Cell    1   Pedido"

![image](https://github.com/user-attachments/assets/64d72ebe-a14d-4b26-8867-d4801c84cff4)

After clicking, the details of the NF should appear as shown in the image below

![image](https://github.com/user-attachments/assets/911b182a-3518-4b67-bb54-a2845ec91384)

**Comment by mardukbp** [comment]
I have to implement a keyword for clicking cells with links. I will do it ASAP. The last two weeks I have been dealing with migraines.

**Comment by mardukbp** [comment]
I just released version 2.11.3 with support for table cells of type link. I tested it with a GuiTree, but I could not test it with a GuiGridView. Please try

```
Select Cell   1   Pedido
```

**Comment by igorquinta** [comment]
@mardukbp It didn't work, now kw can't find the "Pedido" column:
![image](https://github.com/user-attachments/assets/dd2ca96f-f825-4c50-bb8a-1a93af6ba4fc)

![image](https://github.com/user-attachments/assets/72649f03-a47f-42a2-a7c8-348f49d46e19)

**Comment by igorquinta** [comment]
Maybe an argument to select the table could work, but I don't know the effort of that.

**Comment by mardukbp** [comment]
You are right. It's almost done. Next week I will release a new version in which all cell keywords allow specifying the table number.

**Comment by mardukbp** [comment]
I just released version 2.13.1. Now it should work.

**Comment by igorquinta** [comment]
@mardukbp Thank you very much, works! But i used the kw "Select cell" to success.

![image](https://github.com/user-attachments/assets/a28756ef-650b-443f-90c9-c8d6e43e8085)

I can close the issue?

**Comment by mardukbp** [comment]
Perfect! Not yet. I will add that to the documentation and also modify
